.gitattributes

```diff
@@ -0,0 +1,33 @@
+# Define the line ending behavior of the different file extensions
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text text=auto eol=lf
+
+# Denote all files that are truly binary and should not be modified.
+*.png binary
+*.jpg binary
+*.gif binary
+*.jpeg binary
+*.zip binary
+*.phar binary
+*.ttf binary
+*.woff binary
+*.woff2 binary
+*.eot binary
+*.ico binary
+*.mo binary
+*.pdf binary
+*.xsd binary
+*.ts binary
+*.exe binary
+
+# Remove files for archives generated using `git archive`
+codeception.yml export-ignore
```

Comment:
This file should not be excluded, where did you get this from? We also need to fix up that tool

Reply:
Removed https://github.com/spryker-sdk/upgrader/pull/2/commits/c9afde3077cc405f57fd460a5bcf7eb9edf8184c
Copied from there https://github.com/spryker-sdk/brancho/blob/master/.gitattributes